### PR TITLE
[SDFID-388] Retrieve annotation type name for qcode

### PR DESCRIPTION
**Pull request by pablopunk**

### CODE REVIEW

File: scripts/core/editor3/components/annotations/AnnotationPopup.jsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ import {HighlightsPopupPresentation} from '../HighlightsPopupPresentation';
 import {UserAvatar} from 'apps/users/components/UserAvatar';
 
 const Annotation = ({annotation, editorNode, showPopup, highlightId, highlightsManager}) => {
-    const {author, avatar, date, msg, annotationType: type} = annotation.data;
+    const {author, avatar, date, msg, annotationType} = annotation.data;
+    const annotationTypes = ng.get('metadata').values.annotation_types;
```
Comment (pablopunk) [inline scripts/core/editor3/components/annotations/AnnotationPopup.jsx]: @petrjasek is there any way I can avoid using `ng` to retrieve metadata values?

Comment (ioanpocol) [reply scripts/core/editor3/components/annotations/AnnotationPopup.jsx]: We can add it on store when we init it.

Comment (pablopunk) [reply scripts/core/editor3/components/annotations/AnnotationPopup.jsx]: So I should use `ng` when we initialize the store then?

Comment (ioanpocol) [reply scripts/core/editor3/components/annotations/AnnotationPopup.jsx]: Yes, I think it is already used there.

**Pull request merged**